forge/test/model_demos/high_prio/nlp/pytorch/test_whisper_large_v3_turbo.py

```diff
@@ -0,0 +1,39 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
+
+# SPDX-License-Identifier: Apache-2.0
+# Whisper Large V3 turbo - Automatic speech recognition and speech translation
+# Model link : https://huggingface.co/openai/whisper-large-v3-turbo
+# By default, Transformers uses the sequential algorithm.
+# To enable the chunked algorithm, pass the chunk_length_s parameter to the pipeline.
+# For large-v3, a chunk length of 30-seconds is optimal. To activate batching over long audio files, pass the argument batch_size
+
+import torch
+from transformers import AutoModelForSpeechSeq2Seq, AutoProcessor, pipeline
```

Comment:
Can we call the file just test_whisper? 

There are a few test_whisper_*, which should end up in a single test file :))

Reply:
will update it